modules/collectd/manifests/service.pp

```diff
@@ -7,4 +7,11 @@
     ensure  => running,
     restart => "/etc/init.d/collectd restart || { pkill -9 collectd; ps aux | grep '/usr/lib/collectd/file_handles.sh' | awk '{{print \$2}}' | xargs kill -9 ;/etc/init.d/collectd start; }",
   }
+
+  @@icinga::check { "check_collectd_running_${::hostname}":
+    check_command       => 'check_nrpe!check_proc_running!collectd',
+    service_description => 'collectd not running',
```

Comment:
Should the description be for the positive case?

Reply:
Yeah, maybe "collectd system statistics daemon" would be better.

Reply:
I've merged it with this change, but do let me know what you think!